### PR TITLE
Geomesa Quickstart Cassandra

### PR DESCRIPTION
**Pull request by aheyne**
* added cassandra quickstart
* added docs for quickstart
* bumped tutorials version

Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

File: geomesa-quickstart-cassandra/README.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+This tutorial is the fastest and easiest way to get started with GeoMesa.  It is
+a good stepping-stone on the path to the other tutorials that present
+increasingly involved examples of how to use GeoMesa.
+
+In the spirit of keeping things simple, the code in this tutorial only does a
+few small things:
+
+1.  establishes a new (static) SimpleFeatureType
+2.  prepares the Cassandra table to store this type of data
+3.  creates a few hundred example SimpleFeatures
```
Comment (jnh5y) [inline geomesa-quickstart-cassandra/README.md]: It creates 1000 features.  ;)

File: geomesa-quickstart-cassandra/README.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+The only dynamic element in the tutorial is the Cassandra destination; that is
+a property that you provide on the command-line when running the code.
+
+Prerequisites
+-------------
+
+Before you begin, you must have the following:
+
+* an instance of Cassandra 3.x standalone or running on Hadoop 2.2 or better,
```
Comment (jnh5y) [inline geomesa-quickstart-cassandra/README.md]: Do we need Hadoop 2.2?  I don't think so.

File: geomesa-quickstart-cassandra/README.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    Feature ID Observation.990 | Who: Addams | What: 990
+    Feature ID Observation.987 | Who: Addams | What: 987
+
+The quick start code may also be run via Maven using the ``live-test`` profile:
+
+```bash
+$ mvn -pl geomesa-quckstart-cassandra -Plive-test exec:exec -Dcontact_point=<host:port> -Dkeyspace=<keyspace> -Dcatalog_table=<catalog_table> -Dusername=<username> -Dpassword=<password> 
+```
+
+Similarly, the ``username`` and ``password`` are not required here and if you are using a
```
Comment (jnh5y) [inline geomesa-quickstart-cassandra/README.md:137]: I'd pull the directions for creating the namespace into this document.

File: geomesa-quickstart-cassandra/pom.xml [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+            <artifactId>geomesa-cassandra-datastore_${scala.abi.version}</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-utils_${scala.abi.version}</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.datastax.cassandra</groupId>
+            <artifactId>cassandra-driver-core</artifactId>
+            <version>3.0.0</version>
```
Comment (jnh5y) [inline geomesa-quickstart-cassandra/pom.xml]: It is likely worth pulling this out into the root pom.xml as a cassandra.version or cassandra.driver.version

File: geomesa-quickstart-cassandra/pom.xml [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <id>live-test</id>
```
Comment (jnh5y) [inline geomesa-quickstart-cassandra/pom.xml:85]: Cool.  Did you try out the profile?

**Review by jnh5y — APPROVED** [review]
Approved pending a few fixes / questions.

File: geomesa-quickstart-cassandra/README.md (hunk 2 above)
Comment (aheyne) [reply geomesa-quickstart-cassandra/README.md]: Fixed.

File: geomesa-quickstart-cassandra/README.md (hunk 1 above)
Comment (aheyne) [reply geomesa-quickstart-cassandra/README.md]: Technically that's a few hundred :p

File: geomesa-quickstart-cassandra/pom.xml (hunk 5 above)
Comment (aheyne) [reply geomesa-quickstart-cassandra/pom.xml:85]: Yep

File: geomesa-quickstart-cassandra/README.md (hunk 3 above)
Comment (aheyne) [reply geomesa-quickstart-cassandra/README.md:137]: added

File: geomesa-quickstart-cassandra/pom.xml (hunk 4 above)
Comment (aheyne) [reply geomesa-quickstart-cassandra/pom.xml]: moved

**Pull request merged**